### PR TITLE
style(settings): Update styling of settings page

### LINKED ISSUE #464: Elements on settings page appear to float randomly
@spencerbygraves @clagom 

This page needs some design love :) Should the switch align with the banner?

Screenshot 2020-04-27 at 20 26 40

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Refactor styling of the settings page

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/settings
- View new styling

## Details

Fixes #464

Screenshot 2020-05-20 at 12 49 52

### CODE REVIEW

**Comment by barrymcgee** [comment]
Screenshot 2020-05-20 at 15 21 19

File: src/pages/Settings/Settings.js [hunk 1]
```diff
@@ -19,25 +17,27 @@ export default function Settings() {
...
         <div className="settings__toggles">
-          <Notification>
-            You will need to refresh your browser for this change to take effect
-          </Notification>
-          <label className="row">
-            <div className="col-6">Disable Analytics</div>
-            <div className="col-3">
+          <div className="settings__toggles-group">
+            <label
+              onChange={() => {
```
Comment (hatched) [inline src/pages/Settings/Settings.js:24]: I'm not sure why this was moved to the label?

Comment (barrymcgee) [reply src/pages/Settings/Settings.js:24]: This was moved as clicking on the wrapper label toggles the checkbox.. however, it probably better if it's on the input, as it was originally - will revert.

**Comment by hatched** [comment]
Thanks looks much better :)

**Pull request merged**